src/scripts.js

```diff
@@ -11,26 +11,39 @@ import './images/seasoning.png'
 
 import User from './user';
 import Recipe from './recipe';
+import fetchData from './index'
 
-//users
-fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData')
-  .then((response) => response.json())
-  .then(data => generateUser(data))
-  .catch(error => console.log(error.message))
+let userData;
+let recipeData;
+let ingredientsData;
+
+const loadPageHandler = () => {
```

Comment:
If any functionality pops up where we need somehting to be loaded on page, we need to remember that goes in the handler for on load when the function is fired